src/libraries/System.Net.Sockets/tests/FunctionalTests/Connect.cs

```diff
@@ -91,8 +91,7 @@ public async Task Connect_AfterDisconnect_Fails()
         [PlatformSpecific(~(TestPlatforms.OSX | TestPlatforms.FreeBSD))] // Not supported on BSD like OSes.
         public async Task ConnectGetsCanceledByDispose()
         {
-            bool usesApm = UsesApm ||
-                           (this is ConnectTask); // .NET Core ConnectAsync Task API is implemented using Apm
+            bool usesApm = UsesApm;
```

Comment:
The change causes SocketError.OperationAborted instead of ObjectDisposed exception.
This is now the same as the other Task based Socket methods.

(I still need to remove this local)